mmdet/models/backbones/hrnet.py

```diff
@@ -207,20 +207,31 @@ class HRNet(BaseModule):
     arXiv: https://arxiv.org/abs/1904.04514
```

Comment:
The citation style is not correct.
Please refer to https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/backbones/mobilenet_v3.py#L17.